ts/components/core/variables/IOStyles.ts

```diff
@@ -78,6 +78,8 @@ const btnSizeLarge = 56;
 // NEW Design System
 const btnBorderRadius = 8;
 const btnSizeDefault = 48;
+// TODO: Replace the number type with the new IOIconSizeScale
```

Comment:
It might be useful to add a Jira task url to track this TODO. What do you think?

Reply:
All the `Design System` JIRA tasks generally refer to activities that are more macro than micro like this one. More specifically, the `IOIconSizeScale` type is already included in [this PR](https://github.com/pagopa/io-app/pull/4551). If it's merged before this one, we can just update the code and delete the `TODO` completely.